Makefile

```diff
@@ -73,14 +73,7 @@ lint:
 
 .PHONY: vet
 vet:
-	@VETOUT=`$(GOVET) ./... 2>&1`; \
```

Comment:
Why this in the current PR?

Reply:
From PR description:

>Fixed "vet" target in makefile. Previously it was coded to fail if govet produced
any output, which is not a correct assumption. Govet can produce output even in
case of success [1]. The correct way to know if govet is successful is to rely on exit
code, which is exactly what is done now after this change.
[1] - see for example https://travis-ci.org/census-instrumentation/opencensus-service/builds/536550187

Reply:
Please separate in a different PR :)

Reply:
Sure, let me do it.

Reply:
I created a separate PR: https://github.com/census-instrumentation/opencensus-service/pull/558

Please review.